task3-4/task3-4/BirthdayScreenList/ViewWithBirthdays.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ViewWithBirthdays.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 25.09.2022.
+//
+
+import UIKit
+
+/// Представление отображающее информацию о контакте и его дне рождения.
+final class ViewWithBirthDays: UIView {
```

Comment:
```suggestion
final class WithBirthDaysView: UIView {
```
в конце обозначаем то чем является если это лейбл значит в конце Label